pkgs/servers/home-assistant/custom-components/smartir/default.nix

```
@@ -0,0 +1,37 @@
+{ lib
+, buildHomeAssistantComponent
+, fetchFromGitHub
+, home-assistant
```

```suggestion
, home-assistant
, aiofiles
, broadlink
```

We're in the correct callPackages scope, so you can consume dependencies here directly.